app/admin/publication.rb

```diff
@@ -1,14 +1,14 @@
 ActiveAdmin.register Publication do
-  
```

Comment:
Having these changes in this patch makes it difficult to review this change.

Reply:
I understand, but Emacs strips those extra spaces for me.  In fact, there are so many in Muscat that it is difficult for me to stop Emacs to strip them everywhere else.  Shouldn't be a policy in Muscat, like in other projects, to strip them before commiting them?  I know that there are git hooks for that.

Reply:
On the other hand, I have just seen that from the Github web interface it is possible to hide whitespace changes, using the icon in the middle of the **Review changes**  button line.